python3/tests/test_dnf_plugins.py

```diff
@@ -0,0 +1,96 @@
+"""Test module for dnf accesstoken"""
+import unittest
+import sys
+import json
+from unittest.mock import MagicMock, patch
+
+sys.modules["urlgrabber"] = MagicMock()
+
+# Disable wrong import postition as need to mock some sys modules first
+#pylint: disable=wrong-import-position
+
+# Disable unused-argument as some mock obj is not used
+#pylint: disable=unused-argument
+
+# Some test case does not use self
+#pylint: disable=no-self-use
+
+from dnf_plugins import accesstoken
+from dnf_plugins import ptoken
+
+REPO_NAME = "testrepo"
+
+def _mock_repo(a_token=None, p_token=None, baseurl=None):
+    mock_repo = MagicMock()
+    mock_repo.accesstoken = a_token
+    mock_repo.ptoken = p_token
```

Comment:
If there is a good reason to add this "ptoken" config in dnf repo, then what information it would contain?

Reply:
a boolean to indicate whether enable pool token for this repo. refer to https://github.com/xapi-project/xen-api/blob/master/ocaml/xapi/repository_helpers.ml#L432